controller/src/main/java/org/jboss/as/controller/descriptions/ModelDescriptionConstants.java

```diff
@@ -192,6 +193,7 @@
     public static final String FILTERED_OPERATIONS = "filtered-operations";
     public static final String FIXED_PORT = "fixed-port";
     public static final String FIXED_SOURCE_PORT = "fixed-source-port";
+    public static final String FOLDER = "folder";
```

Comment:
Hmm, continuing my debate with myself, I think directory is better. That comment I linked discusses zip file 'folders' but it seems to be in the context of Windows, not as a universal notion.

The zip spec refers to a directory https://pkware.cachefly.net/webdocs/casestudies/APPNOTE.TXT section 4.3.8

Java's ZipEntry class also calls it a directory "isDirectory()"